packages/components/src/styles/components/tabs.scss

```diff
@@ -34,6 +34,7 @@ $hds-tabs-sizes: ( "medium", "large" );
   margin: 0;
   padding: 0;
   overflow-x: auto;
+  isolation: isolate;
```

Comment:
This seems like a reasonable fix to me.

Did you also check the z-index of the .hds-tabs__tab-indicator though?

Code snippet:
```
.hds-tabs__tab-indicator {
  position: absolute;
  right: 0;
  bottom: 0;
  // notice: this custom prop is set dynamically via JavaScript
  left: var(--indicator-left-pos, 0);
  z-index: 10;
```

Reply:
@KristinLBradley what do you mean? that we could lower it (eg. to `1`)?

Reply:
I just wondered if z-index: 10 might be too high? Anyway I think the fix you made is a good solution.

Reply:
Yeah, no no it's a very good point. I am testing with `z-index: 1` and if there are no downsides (I don't think) I'll commit the change.

Reply:
OK, made the change. Thanks again, good spot!